packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -399,12 +411,75 @@ class EditableText extends StatefulWidget {
   /// container will start with enough vertical space for one line and
   /// automatically grow to accomodate additional lines as they are entered.
   ///
-  /// If it is not null, the value must be greater than zero. If it is greater
-  /// than 1, it will take up enough horizontal space to accomodate that number
-  /// of lines.
+  /// If this is not null, the value must be greater than zero, and it will lock
+  /// the input to the given number of lines. Setting [minLines] as well allows
+  /// the input to grow between the indicated range.
+  ///
+  /// The full set of behaviors possible with [minLines] and [maxLines] are as
+  /// follows.
+  ///
+  /// Input that occupies a single line and scrolls horizontally as needed
+  /// ```dart
+  /// TextField()
+  /// ```
+  ///
+  /// Input whose height grows from one line to as many lines as needed for the
+  /// text that was entered
```

Comment:
Should explain that the TextField will keep growing until it reaches the limits imposed by its parent.